### DISCUSSION TITLE
Tokenize

### DISCUSSION THREAD

**Discussion by Om255** [Kab1r/Google-Assistant-Discord-Bot#9] [Q&A] [answered]
Can you please tell clearly how I Tokenize this file to run my bot please Bro teel

**Comment by Kab1r** [comment, marked as answer]
I do not understand what you are asking, what file?

**Reply by Om255** [reply]
Your Goole assistant file

**Reply by Om255** [reply]
Bro your Google code 
Now still you not understand

**Reply by Kab1r** [reply]
You can either put the path to that file in the expected environment variable, or you can copy the json content directly into the environment variable
